website/src/components/Homepage/Header/HomepageHeader.module.scss

```diff
@@ -1,67 +1,76 @@
+@import '@site/src/css/fonts';
+
 .heroBanner {
+  --ifm-heading-color: #fff;
```

Comment:
Why do you overload these variables locally, in `.heroBanner`? Why not in _root.scss?

Reply:
My mistake. I was going to reschedule, but I forgot)